meta-client/src/test/java/org/apache/helix/metaclient/impl/zk/TestZkMetaClient.java

```diff
@@ -504,6 +504,28 @@ public void handleDirectChildChange(String key) throws Exception {
     }
   }
 
+  @Test
+  public void testChangeListener() throws Exception {
+    final String basePath = "/TestZkMetaClient_ChangeListener";
+    final int count = 100;
+    try (ZkMetaClient<String> zkMetaClient = createZkMetaClient()) {
+      zkMetaClient.connect();
+      DataChangeListener listener = new DataChangeListener() {
+
+        @Override
+        public void handleDataChange(String key, Object data, ChangeType changeType)
+            throws Exception {
+        }
+      };
+      zkMetaClient.subscribeDataChange(basePath, listener, false);
```

Comment:
can we also have test case where persistent watcher is not set, and the test still passes.

Reply:
TFTR! Good point. Added.